### PR TITLE
support multiple ssh keys

### PR DESCRIPTION
**Pull request by sudoapt-getclean**
I needed support for multiple keys, the cleanest way to do that seemed to me to just change the existing var.ssh_keys to a list

An alternative implementation that would be backwards compatible would be to introduce a separate variable ssh_keys and merge them in a local. let me know if you prefer that.

### CODE REVIEW

**Comment by yupwei68** [comment]
Hi @sudoapt-getclean , thanks for opening this PR. 
1. I prefer not to break the users by add an extra variable "ssh_keys"
2. Please enabled this feature into fixture/test/main.tf and test if it works when it's assigned and if it works when it's not assigned
3. Please add this feature into the example of README.md

**Review by CrowderKroger — CHANGES_REQUESTED** [review]

File: variables.tf [hunk 1]
```diff
@@ -143,6 +143,12 @@ variable "delete_os_disk_on_termination" {
   default     = false
 }
 
+variable "os_disk_size_gb" {
```
[the remark this answers is not in the document]
Comment (sudoapt-getclean) [reply variables.tf:146]: Yes, that is a mess-up on my side,.. I'm also moving master forward to accommodate my project. I should have created this PR from a stable branch.

File: README.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ More specifically this provisions:
 
 1 - New vnet for all vms
 
-2 - Ubuntu 14.04 Server VMs using `vm_os_publisher`, `vm_os_offer` and `vm_os_sku` which is configured with:
+2 - Ubuntu 18.04 Server VMs using `vm_os_publisher`, `vm_os_offer` and `vm_os_sku` which is configured with:
```
Comment (sudoapt-getclean) [inline README.md:127]: This is an unrelated fix because 18.04 is actually used n the example below

**Review by yupwei68 — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**